### DISCUSSION TITLE
SPRINT 0 - Meet miércoles 28/09/2022 19 hs. Grupo MiniDevs

### DISCUSSION THREAD

**Discussion by veroivk** [FrancoGL/ISPC-FullStack#17] [Daily]
**SPRINT 0**
_Fecha inicio = 17/09/2022 - Fecha Fin = 03/10/2022_

Quinta reunión. 

Trabajamos el mapa del sitio. 
Además conversamos sobre los detalles que harían del sitio web una experiencia diferente.
La profesora Viviana Guillén nos comentó que ya existe el sitio de la secretaria de turismo de la provincia de Córdoba y entonces, ver que podríamos brindar como un bonus.

**Comment by ZafkielEdt** [comment]
Como bonus se pensó en ofrecer paquetes de destinos.
